packages/in_app_purchase/in_app_purchase_android/android/src/main/java/io/flutter/plugins/inapppurchase/InAppPurchasePlugin.java

```diff
@@ -32,16 +32,15 @@ static final class MethodNames {
         "BillingClient#startConnection(BillingClientStateListener)";
     static final String END_CONNECTION = "BillingClient#endConnection()";
     static final String ON_DISCONNECT = "BillingClientStateListener#onBillingServiceDisconnected()";
-    static final String QUERY_SKU_DETAILS =
-        "BillingClient#querySkuDetailsAsync(SkuDetailsParams, SkuDetailsResponseListener)";
+    static final String QUERY_PRODUCT_DETAILS =
+        "BillingClient#queryProductDetailsAsync(QueryProductDetailsParams, ProductDetailsResponseListener)";
     static final String LAUNCH_BILLING_FLOW =
         "BillingClient#launchBillingFlow(Activity, BillingFlowParams)";
     static final String ON_PURCHASES_UPDATED =
         "PurchasesUpdatedListener#onPurchasesUpdated(int, List<Purchase>)";
-    static final String QUERY_PURCHASES = "BillingClient#queryPurchases(String)";
     static final String QUERY_PURCHASES_ASYNC = "BillingClient#queryPurchasesAsync(String)";
     static final String QUERY_PURCHASE_HISTORY_ASYNC =
-        "BillingClient#queryPurchaseHistoryAsync(String, PurchaseHistoryResponseListener)";
+        "BillingClient#queryPurchaseHistoryAsync(String)";
```

Comment:
Nit/question to better my understanding:
I see this method name was changed to omit the response listener, even though the method we are using now still has a [response listener](https://developer.android.com/reference/com/android/billingclient/api/BillingClient#queryPurchaseHistoryAsync(com.android.billingclient.api.QueryPurchaseHistoryParams,%20com.android.billingclient.api.PurchaseHistoryResponseListener)). Is this because we aren't passing it it in from dart, and this signature is meant to represent what arguments are being passed in the map method invocation?

If so, should the above  `BillingClient#queryProductDetailsAsync(QueryProductDetailsParams, ProductDetailsResponseListener)` have the listener removed from its signature (as I believe you are also not passing in that listener but rather creating it when you invoke the native method in MethodCallHandlerImpl), or if not should this be changed back to include the listener in the signature?

Reply:
Thank you for the review!

I looked into the convention that we are using in the plugin regarding method naming. It seems that we **do** mention all parameters that get passed in the original method in `BillingClient`. In this case, that would include `PurchaseHistoryResponseListener`.

I made sure to update the existing entries as well, so they are all aligned.